lib/journaled.rb

```diff
@@ -16,7 +16,19 @@ module Journaled
   mattr_accessor(:http_open_timeout) { 2 }
   mattr_accessor(:http_read_timeout) { 60 }
   mattr_accessor(:job_base_class_name) { 'ActiveJob::Base' }
-  mattr_accessor(:transactional_batching_enabled) { true }
+  mattr_writer(:transactional_batching_enabled) { true }
+
+  def self.transactional_batching_enabled?
+    Thread.current[:journaled_transactional_batching_enabled] || @@transactional_batching_enabled
```

Comment:
this might be overly cautious, but it seems risky that someone could accidentally consume `Journaled.transactional_batching_enabled` instead of `Journaled.transactional_batching_enabled?`, and get a different result (which would probably be surprising). Should we name the global setting something more explicit so that it's harder to misuse?

Reply:
There shouldn't be an attr_reader for it, just an attr_writer, and I made sure I saw tests failing when I renamed it to have the ?

Reply:
good point! I didn't consider the accessor -> writer change